gcamreader/cli.py

```diff
@@ -0,0 +1,161 @@
+from pathlib import Path
+from .querymi import LocalDBConn, RemoteDBConn, parse_batch_query
+import click
+from click_default_group import DefaultGroup
+import sys
+
+
+@click.group(
+    cls=DefaultGroup,
+    default="local",
+    default_if_no_args=True,
+)
+@click.version_option()
+def cli():
+    """
+    Run queries against a gcam scenario database.
+
+        Saves outputs as .csv
+
+    Documentation: https://github.com/JGCRI/gcamreader/
+    """
+
+
+@cli.command(name="local")
+@click.option(
+    "-d",
+    "--database_path",
+    type=click.Path(exists=True, file_okay=False, readable=True, path_type=Path),
+    required=True,
+    help="path to database file (i.e. parent of *.basex dir)",
+)
+@click.option(
+    "-q",
+    "--query_path",
+    type=click.Path(exists=True, dir_okay=False, readable=True, path_type=Path),
+    required=True,
+    help="path to xml with queries to run (i.e: Main_queries.xml)",
+)
+@click.option(
+    "-o",
+    "--output_path",
+    type=click.Path(exists=True, file_okay=False, writable=True, path_type=Path),
+    help="path to output (i.e. where .csv files should be created)",
+)
+def local(database_path: Path, query_path: Path, output_path: Path):
+    """
+    query gcam scenario databases
+    """
+
+    print(f"opening: {database_path.absolute()}", file=sys.stderr)
+    if not list(database_path.glob("*.basex")):
+        print(f"basex files missing: {database_path}", file=sys.stderr)
```

Comment:
is there a reason to print to stderr directly rather than use `click.echo('message', err=True)`?